pkg/netutil/netutil.go

```diff
@@ -21,7 +22,7 @@ func IsPortActive(port string) bool {
 		return false
 	}
 
-	conn, err := net.Dial("tcp", dockerIP+":"+port)
+	conn, err := net.DialTimeout("tcp", dockerIP+":"+port, 1*time.Second)
```

Comment:
@marvinhinz, can you experiment with network here? Replace `tcp` with `tcp4`

```suggestion
	conn, err := net.Dial("tcp4", dockerIP+":"+port)
```

We already use IPv4 only in some other places, but not here:

- #6593

Reply:
P.S. From https://pkg.go.dev/net#Dial

It may fail because we don't pass IPv6 correctly (but we don't need it anyway?):

> If the host is a literal IPv6 address it must be enclosed in square brackets, as in "[2001:db8::1]:80" or "[fe80::1%zone]:80".

Reply:
I didn't end up doing this, but it can still be done. I experimented with it at one point but it wasn't really relevant it didn't seem.